src/Enumerators/Catagories.java

```diff
@@ -8,5 +8,28 @@
  *
  */
 public enum Catagories {
```

Comment:
There's a lot of added complexity in our role assignment. We should focus first on making a simple one that works